### PR TITLE
Add standard

### PR DESCRIPTION
**Pull request by fanatid**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+  - "0.10"
+  - "0.12"
+  - "4"
+  - "5"
+  - "6"
+  - "7"
+matrix:
+  include:
+    - node_js: "6"
+      env: TEST_SUITE=standard
```
Comment (lgeiger) [inline .travis.yml]: This should be
```yaml
env: TEST_SUITE=lint
```

File: package.json [hunk 2]
```diff
@@ -24,6 +26,7 @@
   },
   "homepage": "https://github.com/lgeiger/node-abi#readme",
   "devDependencies": {
+    "standard": "*",
```
Comment (lgeiger) [inline package.json]: I'd prefer using `^8.6.0` instead of `*`.

We have greenkeeper enabled for this repo which will keep everything uptdate.

Comment (fanatid) [inline package.json]: I understand you here, I also set `standard` to `^a.b.c` a few month ago..
but we don't really need this

**Comment by lgeiger** [comment]
Thanks for you're PRs would you like to help maintaining this package?

**Comment by fanatid** [comment]
Thank you, but I would like do more significant improvements before I will be added to maintainers :))

**Pull request merged**